screens/ReservationCalendarScreen.js

```diff
@@ -0,0 +1,508 @@
+import React from 'react'
+import {View, Text, ScrollView, TouchableOpacity, Alert} from 'react-native'
+import FontAwesomeIcon from 'react-native-vector-icons/FontAwesome'
+import MCIcon from 'react-native-vector-icons/MaterialCommunityIcons'
+import styles from '../styles'
+import {LocaleContext} from '../locales/LocaleContext'
+import ScreenHeader from "../components/ScreenHeader";
+import MenuButton from "../components/MenuButton";
+import {withContext} from "../helpers/contextHelper";
+import {ThemeScrollView} from "../components/ThemeScrollView";
+import {ThemeContainer} from "../components/ThemeContainer";
+import {Calendar, LocaleConfig, Agenda, AgendaList} from 'react-native-calendars';
+import {api, dispatchFetchRequest} from '../constants/Backend'
+import {NavigationEvents} from 'react-navigation'
+import TimeZoneService from "../helpers/TimeZoneService";
+import moment from "moment-timezone";
+import {StyledText} from '../components/StyledText'
+import {RenderAgenda, DayCalendar, ReservationDayEvent, ReservationEvent} from "../components/Calendars";
+import AddBtn from '../components/AddBtn'
+import Modal from 'react-native-modal';
+import {OptionModal} from "../components/OptionModal";
+import {MainActionButton} from '../components/ActionButtons'
+import SegmentedControlTab from "react-native-segmented-control-tab";
+import {CheckBox} from 'react-native-elements'
+import Icon from 'react-native-vector-icons/Ionicons'
+import {CalendarEvent} from "../components/CalendarEvent";
+import {Ionicons} from '@expo/vector-icons';
+import {compose} from "redux";
+import {connect} from 'react-redux'
+import {MaterialCommunityIcons} from '@expo/vector-icons';
+import {getTableLayouts, getTableLayout} from '../actions'
+
+class CalendarScreen extends React.Component {
```

Comment:
Can we change the class name to match the file name ReservationCalendarsScreen?

Reply:
Ok, forgot to change name